spec/factories/addresses.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :address do
+    home_address { Faker::Address.street_address }
+    zip_code { Faker::Address.zip_code }
+    city { create(:city) }
+    user { create(:user) }
```

Comment:
```suggestion
    city
    user
```

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#implicit-definition